gitopia/chain.json

```diff
@@ -392,6 +402,9 @@
         "provider": "StakeAngle"
       },
       {
+        "address": "https://gitopia.rpc.m.anode.team",
+        "provider": "AlxVoy ⚡ ANODE.TEAM"
+      },
```

Comment:
oh, you didn't seem to bracket things correctly, and the chain.json validation fails on this